### DISCUSSION TITLE
Download font family easier with tools like `DownGit`

### DISCUSSION THREAD

**Discussion by KiruyaMomochi** [ryanoasis/nerd-fonts#811] [Ideas]
**Try it** -> https://downgit.github.io/#/home?url=https://github.com/ryanoasis/nerd-fonts/tree/master/patched-fonts/FiraCode

Currently it's hard to download a Nerd font family following README, because **I must clone the full repo** if I don't want to use package manager.

However, it may be possible to download files from web or in shell script, following these steps:
1. Get the link to directory of font family
2. Use GitHub API to iterator the directory and download all files
3. For client: Install fonts. For web: create a zip file with jszip.

See [Download a single folder or directory from a GitHub repo](https://stackoverflow.com/questions/7106012/download-a-single-folder-or-directory-from-a-github-repo).

**Comment by Finii** [comment]
What is wrong with the Release Page? https://github.com/ryanoasis/nerd-fonts/releases

Each (patched) font family in its own nice zip?
![image](https://user-images.githubusercontent.com/16012374/159185398-4f553ea1-0a19-46e2-a2fc-d77fae4d26d3.png)


For example (for your Fira Code): https://github.com/ryanoasis/nerd-fonts/releases/download/2.2.0-RC/FiraCode.zip

**Reply by KiruyaMomochi** [reply]
Sorry for my bad.

I thought these zips are outdated before due to release date is `22 Nov 2021`. Now I have downloaded these files and found they are the latest.

These zips work nice, thanks, and sorry for your inconvenience.

**Reply by Finii** [reply]
The RC is kind of a rolling release, every time some fix is applied to master the release is automatically updated.

Glad it works for you.
